### DISCUSSION TITLE
User authorization problem when starting localhost.

### DISCUSSION THREAD

**Discussion by xxx25xxx** [iTwin/itwinjs-core#3250] [Q&A]
Hi, I've got a problem similar to https://github.com/iTwin/itwinjs-core/discussions/2965, my scopes are 
IMJS_AUTH_CLIENT_SCOPES ="email openid profile organization itwinjs imodels:read", no chance to make it running. Any help is appreciated, thanks !
[js](https://user-images.githubusercontent.com/100188646/155079452-eb96c1c0-cd43-43fd-bf81-d9c8ee9a9f78.jpg)

**Comment by aruniverse** [comment]
Please provide more information. What versions of iTwin.js and itwin/web-viewer-react are you using?
